test/topology_experimental_raft/test_topology_ops.py

```diff
@@ -26,9 +26,10 @@ async def test_topology_ops_without_tablets(request, manager: ManagerClient):
     await run_test_topology_ops(request, manager, {'experimental_features': ['consistent-topology-changes']})
 
 
-@pytest.mark.asyncio
-async def test_topology_ops_with_tablets(request, manager: ManagerClient):
-    await run_test_topology_ops(request, manager, {'experimental_features': ['consistent-topology-changes', 'tablets']})
+# FIXME: the test fails because of #17025. We need to re-enable it once this issue is fixed.
```

Comment:
cc @tgrabiec @bhalevy

Reply:
@patjed41 maybe we should instead reenable it and make it `xfail`?

We could also run the test in tablets mode just without the background writes.

Reply:
I realized there is something like `@pytest.mark.parametrize`. I think I should use it instead of creating two new functions. I don't think we can `xfail` only one ​parameterization, so I'll disable background writes for tablets.

Reply:
We should leave in place some incentive to re-enable background writes in tablets mode and re-test once https://github.com/scylladb/scylladb/issues/17025 is fixed.

A comment in https://github.com/scylladb/scylladb/issues/17025 could be one thing, but there's no guarantee that whoever fixes the issue eventually will address it.

Perhaps another issue -- reeneable background writes in tablets mode?

Reply:
Sounds good, I'll open it.